personManagerAllocatedEventDefinition.json

```diff
@@ -0,0 +1,15 @@
+{
+  "eventType": "PERSON_MANAGER_ALLOCATED",
```

Comment:
I noticed in the ADR event type was lower case and `.` separated, does this need to be changed to reflect that? is that the preferred naming strategy for event types?

Reply:
I think so yes - the probation data events were originally created in this format (uppercase with underscores), however this was re-defined for HMPPS domain events.  @mxco86 might be able to provide a link to the message spec in Confluence.

Reply:
```suggestion
  "eventType": "person.manager.allocated",
```
as a first pass for an alternative